Cargo.toml

```diff
@@ -44,10 +44,10 @@ log = { version = "0.4.8", features = ["max_level_info", "release_max_level_info
 mime = "0.3"
 num_cpus = "1"
 # must match what's used by googleapis-raw
-protobuf = "2.15"
+protobuf = "2.16.2"
 rand = "0.7"
 regex = "1.3"
-sentry = { version = "0.18", features = ["with_curl_transport"] }
+sentry = { version = "0.19", features = ["debug-logs", "with_curl_transport"] }
```

Comment:
It looks like there's an in-progress upgrade to sentry 0.19 in this PR. That seems unexpected.

Reply:
it is, but again, can't get this running locally without it. Local builds fail with
```
   Compiling sentry-types v0.14.1
error[E0659]: `debugid` is ambiguous (name vs any other name during import resolution)
  --> /home/jrconlin/.cargo/registry/src/github.com-1ecc6299db9ec823/sentry-types-0.14.1/src/protocol/v7.rs:18:5
   |
18 | use debugid::DebugId;
   |     ^^^^^^^ ambiguous name
   |
   = note: `debugid` could refer to a crate passed with `--extern`
   = help: use `::debugid` to refer to this crate unambiguously
note: `debugid` could also refer to the module defined here
  --> /home/jrconlin/.cargo/registry/src/github.com-1ecc6299db9ec823/sentry-types-0.14.1/src/protocol/v7.rs:41:1
   |
41 | / pub mod debugid {
42 | |     pub use debugid::{BreakpadFormat, DebugId, ParseDebugIdError};
43 | | }
   | |_^
   = help: use `self::debugid` to refer to this module unambiguously

error: aborting due to previous error
```